### DISCUSSION TITLE
part-ii/2019-99/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#308] [Question comments]
Part II 2019 Number Theory: Paper 4, Section I, I 

https://questions.tripos.org/part-ii/2019-99/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

Bookwork! See near the start of lecture 14 for the proof. Prove the product diverges, then take the logarithm of (the partial version of) the sum and use the Taylor series expansion of $\log(1+x)$ to show the sum diverges.

</details>
